### PR TITLE
Fixed compilation for dpcpp

### PR DESCRIPTION
**Pull request by ravil-mobile**

### CODE REVIEW

File: src/Parallel/AcceleratorDevice.cpp [hunk 1]
```diff
@@ -9,16 +9,24 @@ void AcceleratorDevice::bindSyclDevice(int deviceId) {
   std::ostringstream info;
 
   try {
+#ifdef __DPCPP_COMPILER
+    syclDevice = sycl::device(sycl::gpu_selector_v);
+#else
```
Comment (sebwolf-de) [inline src/Parallel/AcceleratorDevice.cpp:14]: Wouldn't another `#elif` be sufficient here?

**Review by sebwolf-de — APPROVED** [review]
Hi, generally these changes look good to me, just at one point, IMHO code could be made easier.

File: src/Parallel/AcceleratorDevice.cpp (hunk 1 above)
Comment (ravil-mobile) [reply src/Parallel/AcceleratorDevice.cpp:14]: done

**Comment by ravil-mobile** [comment]
Hi @sebwolf-de , thanks for reviewing the code!

**Review by sebwolf-de — APPROVED** [review]

**Pull request closed without merging**